test/lib/jdk/test/lib/Asserts.java

```diff
@@ -404,50 +403,49 @@ public static <T extends Comparable<T>> void assertGreaterThan(T lhs, T rhs, Str
     /**
      * Shorthand for {@link #assertNotEquals(Object, Object)}.
      *
-     * @param lhs The left hand side of the comparison.
-     * @param rhs The right hand side of the comparison.
+     * @param unexpected The unexpected value
+     * @param actual The actual value
      * @see #assertNotEquals(Object, Object)
      */
-    public static void assertNE(Object lhs, Object rhs) {
-        assertNotEquals(lhs, rhs);
+    public static void assertNE(Object unexpected, Object actual) {
+        assertNotEquals(unexpected, actual);
     }
 
     /**
      * Shorthand for {@link #assertNotEquals(Object, Object, String)}.
      *
-     * @param lhs The left hand side of the comparison.
-     * @param rhs The right hand side of the comparison.
+     * @param unexpected The unexpected value
+     * @param actual The actual value
      * @param msg A description of the assumption; {@code null} for a default message.
      * @see #assertNotEquals(Object, Object, String)
      */
-    public static void assertNE(Object lhs, Object rhs, String msg) {
-        assertNotEquals(lhs, rhs, msg);
+    public static void assertNE(Object unexpected, Object actual, String msg) {
+        assertNotEquals(unexpected, actual, msg);
     }
 
     /**
      * Calls {@link #assertNotEquals(Object, Object, String)} with a default message.
      *
-     * @param lhs The left hand side of the comparison.
-     * @param rhs The right hand side of the comparison.
+     * @param unexpected The unexpected value
+     * @param actual The actual value
      * @see #assertNotEquals(Object, Object, String)
      */
-    public static void assertNotEquals(Object lhs, Object rhs) {
-        assertNotEquals(lhs, rhs, null);
+    public static void assertNotEquals(Object unexpected, Object actual) {
+        assertNotEquals(unexpected, actual, null);
     }
 
     /**
-     * Asserts that {@code lhs} is not equal to {@code rhs}.
+     * Asserts that {@code actual} is not equal to {@code unexpected}.
      *
-     * @param lhs The left hand side of the comparison.
-     * @param rhs The right hand side of the comparison.
+     * @param unexpected The unexpected value
+     * @param actual The actual value
      * @param msg A description of the assumption; {@code null} for a default message.
      * @throws RuntimeException if the assertion is not true.
      */
-    public static void assertNotEquals(Object lhs, Object rhs, String msg) {
-        if ((lhs == rhs) || (lhs != null && lhs.equals(rhs))) {
+    public static void assertNotEquals(Object unexpected, Object actual, String msg) {
+        if ((unexpected == actual) || (unexpected != null && unexpected.equals(actual))) {
             msg = Objects.toString(msg, "assertNotEquals")
-                    + ": expected " + Objects.toString(lhs)
-                    + " to not equal " + Objects.toString(rhs);
+                    + ": expected not equals but was " + Objects.toString(actual);
```

Comment:
Given that the comparison is based on Object.equals(...), there may be (rare) cases where `a.equals(b)` returns true but `a.toString()` produces different output from `b.toString()`? To address all cases, does it make sense to compare the result of both toString() and print both in case that they are different?

Reply:
I don't think this is worth doing. See my previous comment at https://github.com/openjdk/jdk/pull/21101#discussion_r1888624756.

Reply:
Alright, probably won't affect majority of cases. Can be confusing for the rare corner cases though.